_posts/2019-03-22-Episode-17-TensorFlow.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: Episode 17 - TensorFlow - Open Source Directions
+---
+
+<iframe width="1349" height="528" src="https://www.youtube.com/embed/-fiEzhva2ZY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Introductions
+
+Welcome to Open Source Directions, the webinar that informs you about the future of open source projects.
+This episode, the project being highlighted is [TensorFlow](https://www.tensorflow.org/)
+
+* **Host:** Anthony Scopatz
+* **Co-Host:** David Charboneau
```

Comment:
@bsodenkamp - please change the template to Carol :stuck_out_tongue_winking_eye: